truenas/webui
# NAS-112741 / 22.02 / Removed welcome dialog and added "Help" widget on dashboard (by RehanY147)

Automatic cherry-pick failed. Please resolve conflicts by running:

    git cherry-pick -x 4ec06721183369136de498db4434cd41f197ae5c
    git cherry-pick -x adb8f43c01ab38a3560da15776e0c25c8875fae0
    git cherry-pick -x 8ca4f720daa0321edc5a6932b342d29eacba104f
    git cherry-pick -x 2f531bae79d3a2e5dbc72f7dda00d626f22cd194
    git cherry-pick -x 3f850168d8fe2e59ff591e12fee9811ccd774403
    git cherry-pick -x ce047c91f7910226240abe89a7e922ac813f9053
    git cherry-pick -x 03037bad5b1085b879ecc0366732d1cfe90b28ff

To test
- Make sure that the help widget shows up by default on the dashboard page
- Make sure that it's configurable and hides when unchecked on the Dashboard Configure form
- Check responsive on phone screens
**Note: Also review reorder function of the help widget the same as others.**

Original PR: https://github.com/truenas/webui/pull/6308
Jira URL: https://jira.ixsystems.com/browse/NAS-112741
